BootloaderCorePkg/Stage1A/Ia32/SecEntry.nasm

```diff
@@ -61,7 +63,11 @@ FspApiSuccess:
         add     esp, dword [ASM_PFX(PcdGet32(PcdStage1StackSize))]
         mov     eax, esp
 
+        movd    ebx, mm0
```

Comment:
I did put a comment in previous commit.  Can we remove line 64  and then do  below ?
movd  eax, mm0
push   eax

Reply:
Done!